test/_init_tests.js

```diff
@@ -377,7 +377,10 @@ beforeEach(function() {
   this.timeout(BEFORE_AFTER_TIMEOUT);
   beforeTest();
   testName = this.currentTest.fullTitle();
-  stubConsoleInfoLogWarn();
+  const {verboseLogging} = window.__karma__.config;
+  if (!verboseLogging) {
+    stubConsoleInfoLogWarn();
+  }
```

Comment:
Seems like we're no longer creating the sandbox when `--verbose` is used, resulting in the following error:

```
  TypeError: Cannot read property 'restore' of undefined
      at restoreConsoleInfoLogWarn (/tmp/test/_init_tests.js:373:28 <- /tmp/20fafa07a5f318252b366d0a8daf1740.browserify:72980:29)
      at Context.<anonymous> (/tmp/test/_init_tests.js:410:2 <- /tmp/20fafa07a5f318252b366d0a8daf1740.browserify:73018:3)
```

Reply:
To fix, we'd need to include this change too:

```
// Used to restore info, log, and warn level logging after each test.
function restoreConsoleInfoLogWarn() {
  if (consoleInfoLogWarnSandbox) {
    consoleInfoLogWarnSandbox.restore();
  }
}
```